behaviour/graql/language/match.feature

```diff
@@ -123,6 +123,74 @@ Feature: Graql Match Clause
       | THI |
 
 
+  Scenario: `sub` can be used to retrieve all instances of types that are subtypes of a given type
```

Comment:
On line 53 of this file you have a heading `SCHEMA CONCEPTS` -- i think this would better be `schema queries` now

Reply:
Updated to `schema queries`